examples/protonect/CMakeLists.txt

```diff
@@ -166,6 +166,28 @@ IF(ENABLE_OPENCL)
     LIST(APPEND RESOURCES
       src/opencl_depth_packet_processor.cl
     )
+
+    INCLUDE(CheckCSourceRuns)
```

Comment:
It's OK. The problem is ocl-icd may be entirely different for binary package builders and users. This is only compile-time detection for the ocl-icd bug.

I think it may be fine to disable exception for all users, like [chromium](https://www.chromium.org/chromium-os/build/c-exception-support). Disabling exceptions has some benefits and seems harmless.